### PR TITLE
Kimhieu vuongho555 roman2arabic

### PR DESCRIPTION
**Pull request by trankimhieu**
Finish roman2arabic
Coded by vuongho555

### CODE REVIEW

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 1]
```diff
@@ -3,4 +3,42 @@ class RomanNumeral
   # code code code
   # what? why?
 
-end
\ No newline at end of file
+  RRN1 = {
```
Comment (joneslee85) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: Is it unused?

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 2]
```diff
@@ -3,4 +3,42 @@ class RomanNumeral
...
+    'C' => 100,
+    'D' => 500,
+    'M' => 1000
+  }
+
+  def initialize(number)
+    @r_number = number
+  end
+
+  def to_arabic
```
Comment (joneslee85) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: :+1:

File: roman_to_arabic_numeral/roman_to_arabic.rb (hunk 1 above)
Comment (vuongho555) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: yes, It is redundant :)

File: roman_to_arabic_numeral/roman_to_arabic_spec.rb [hunk 3]
```diff
@@ -2,8 +2,7 @@
 require 'minitest/autorun'
 
 describe RomanNumeral do
-
-  it 'converts Roman numerals to Arabic numerals' do
+  it 'converts Arabic numerals to Roman numerals' do
```
Comment (joneslee85) [inline roman_to_arabic_numeral/roman_to_arabic_spec.rb]: should be other way around

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 4]
```diff
@@ -3,4 +3,42 @@ class RomanNumeral
...
+  {
+    'I' => 1,
+    'V' => 5,
+    'X' => 10,
+    'L' => 50,
+    'C' => 100,
+    'D' => 500,
+    'M' => 1000
+  }
+
```
Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: RRN2 = RRN1.invert

Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: Also RRN is obscure, what about ROMAN_MAPPING  & ARABIC_MAPPING? naming convention

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 5]
```diff
@@ -3,4 +3,42 @@ class RomanNumeral
...
+  def initialize(number)
+    @r_number = number
```
Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: @r_number is obscure too, what about @roman_number ?

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 6]
```diff
@@ -3,4 +3,42 @@ class RomanNumeral
...
+  end
+
+  def to_arabic
+    array = @r_number.split('')
+    array.map!{|a| RRN2[a]}
+    size = array.size - 1
```
Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: can combine into 1 line
```
array = @r_number.split('').map { |x| RRN2[x] }
```

Comment (vuongho555) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: We were impatient. We have been meaning to refactor all lines of code :)

**Pull request merged**